courses/task_seven/seven.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"fmt"
+	"log"
+)
+
+type VehicleError string
+
+func (e VehicleError) Error() string {
+	return string(e)
+}
+
+type Vehicle interface {
+	Move(distance float64) interface{}
+	TankUp(quantity float64) interface{}
+}
+
+type Auto struct {
+	CarBrand        string
+	FuelLevel       float64
+	FuelConsumption float64
+	MaxFuelLevel    float64
+}
+
+func (auto *Auto) Move(distance float64) interface{} {
```

Comment:
```suggestion
func (auto *Auto) Move(distance float64) error {
```
It's not advised to use interface to return value/error in go unlike in PHP